app/layout.tsx

```diff
@@ -9,7 +9,7 @@ export const metadata: Metadata = {
   title: "Starknet Quest",
   description:
     "Starknet Quest helps protocols attract and retain users by creating gamified quest experiences on Starknet.",
-  metadataBase: new URL(process.env.NEXT_PUBLIC_APP_LINK as string),
+  // metadataBase: new URL(process.env.NEXT_PUBLIC_APP_LINK as string),
```

Comment:
please don't comment this (you can add `NEXT_PUBLIC_APP_LINK =https://starknet.quest/` to make it work